modules/apps/change-tracking/change-tracking-service/src/main/java/com/liferay/change/tracking/service/impl/CTScoreLocalServiceImpl.java

```diff
@@ -255,6 +268,24 @@ private CTScore _updateScore(
 
 		ctScore.setScore(score);
 
+		CTCollection ctCollection =
```

Comment:
@gislayne-vitorino can you move this code to a model listener instead and also ensure the transaction doesn't fail if the notification fails to send.